setup.md

```diff
@@ -45,21 +45,17 @@ If you installed Anaconda, you can launch a notebook in two ways:
 >
 > 1. Launch Anaconda Navigator.
 > It might ask you if you'd like to send anonymized usage information to Anaconda developers:
-> ![Anaconda Navigator first launch]({{ page.root }}{% link
-> fig/anaconda-navigator-first-launch.png %})
+> ![Anaconda Navigator first launch]({{ page.root }}{% link fig/anaconda-navigator-first-launch.png %})
 > Make your choice and click "Ok, and don't show again" button.
```

Comment:
These lines can be broken just inside the opening parentheses to get them under 100 characters. I am not sure how strongly people feel about doing this in such cases, but it should work to do:
```
> ![Anaconda Navigator first launch](
{{ page.root }}{% link fig/anaconda-navigator-first-launch.png %})
> Make your choice and click "Ok, and don't show again" button.
```

Reply:
I think this is fixed now. Thanks!